generators/generator-bot-adaptive/package.json

```diff
@@ -20,7 +20,8 @@
     "url": "https://github.com/Microsoft/botframework-components.git"
   },
   "scripts": {
-    "lint": "eslint -c .eslintrc.json ."
+    "lint": "eslint -c .eslintrc.json .",
+    "test": "mocha tests"
```

Comment:
Small note, but `mocha` convention defaults to `test`. If it's not too annoying, the `tests` directory should be named `test` and then this command can simply be `mocha`.

Reply:
You may need to add something like: 
```json
{
  ...,
  "mocha": {
    "recursive": true
  },
  ...
}
```
to properly discover all the tests as well. I'd also suggest adding `checkLeaks`, `exit`, `forbidOnly`, and `forbidPending`.